### ISSUE TITLE
[Suggestion] Different choices for difficulty change

### ISSUE THREAD

**Issue by Krigeris** [sinkillerj/ProjectE#1762]
Our current option for changing difficulty is changing how much emc you get per item burned, but getting say half the EMC for an item, kind of breaks the concept of "equivalent exchange". The old EE, used to have a file you could edit to change the EMC generation rates from the collectors/anti-matter relays/other machines, so you could still have equivalent exchange, but machines would gain EMC at a percentage of the rate. 

Currently the only means of emulating this is by using red stone torches for lesser light, and since the machines give light themselves, it doesn't really function as a proper nerf.

**Comment by Hessin87** [comment]
fair point, I'd also like to have a configurable setting for how much EMC is consumed by the various gadgets, (life,body,soul) stones, more expensive swrg flight, EMC consumed for using charged DM and RM tools, various gimping options for gem armor like either expending EMC for the various features or disabling those recipes all together.

**Comment by Arcami** [comment]
I would like too to change the rate energy collectors generate energy from thin air. As far as I've seen, there's currently now way of doing that.
